hyperspace/testsuite/Cargo.toml

```diff
@@ -21,17 +21,27 @@ ibc-proto = { path = "../../ibc/proto" }
 tendermint-proto = { git = "https://github.com/composableFi/tendermint-rs", rev = "5a74e0f8da4d3dab83cc04b5f1363b018cf3d9e8" }
 
 hyperspace-core = { path = "../core", features = ["testing"] }
+hyperspace-parachain = { path = "../parachain", features = ["testing"] }
 hyperspace-primitives = { path = "../primitives", features = ["testing"] }
```

Comment:
Let's use the `build-metadata-from-ws` feature for tests.

Reply:
The CI fails because of this due to a connection error...